vita3k/gui/src/pkg_install_dialog.cpp

```diff
@@ -30,19 +30,29 @@
 
 namespace gui {
 
-static std::filesystem::path pkg_path = "";
-static std::filesystem::path license_path = "";
-static std::string state, title, zRIF;
-static bool draw_file_dialog = true;
-static bool delete_pkg_file, delete_license_file;
-
 void draw_pkg_install_dialog(GuiState &gui, EmuEnvState &emuenv) {
     host::dialog::filesystem::Result result = host::dialog::filesystem::Result::CANCEL;
     static std::atomic<float> progress(0);
     static std::mutex install_mutex;
     static const auto progress_callback = [&](float updated_progress) {
         progress = updated_progress;
     };
+    static std::filesystem::path pkg_path = "";
+    static std::filesystem::path license_path = "";
+    static std::string title, zRIF;
+    static bool draw_file_dialog = true;
+    static bool delete_pkg_file, delete_license_file;
```

Comment:
Same, it looks like delete_pkg_file and delete_license_file have no need to be static.